### PR TITLE
added a new template

### PR DESCRIPTION
**Pull request by HassanShanJava**
-Followed instructions from the contributing.md
-Added a new portfolio sample on the homepage
-Added target="_blank" on all links on the homepage that way users can open multiple samples instead of being redirected
-Sample is responsive for mobile, tablet, and laptop screens

Images of the work done:
homepage:
porfolio sample

site look:
portfoilio look

### CODE REVIEW

**Comment by mpotane** [comment]
I think, you need to remove the liveserver config from your commit.

**Comment by HassanShanJava** [comment]
How come, could you explain?

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (mpotane) [inline .vscode/settings.json:3]: This one bro.

Comment (HassanShanJava) [reply .vscode/settings.json]: Alright, i just removed it and push it, plz check

**Comment by HassanShanJava** [comment]
Alright, i just removed it and push it, plz check

**Comment by HassanShanJava** [comment]
Hi, could you merge my pull request if there is no problem with the code now

**Review by jofftiquez — APPROVED** [review]

**Pull request merged**